src/objectTranslations/Relationship__c-nl_NL.objectTranslation

```diff
@@ -0,0 +1,166 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomObjectTranslation xmlns="http://soap.sforce.com/2006/04/metadata">
+    <caseValues>
+        <plural>false</plural>
+        <value><!-- Relationship --></value>
+    </caseValues>
+    <caseValues>
+        <plural>true</plural>
+        <value><!-- Relationship --></value>
+    </caseValues>
+    <fields>
+        <help>Gerelateerde Contactpersonen is dit Contact&apos;s_______</help>
+        <label>Contactpersoon</label>
+        <name>Contact__c</name>
+        <relationshipLabel>Relaties</relationshipLabel>
+    </fields>
+    <fields>
+        <help>Extra informatie over deze Relatie</help>
+        <label>Beschrijving</label>
+        <name>Description__c</name>
+    </fields>
+    <fields>
+        <help>Systeem veld die de twee kanten van de relatie samen linkt. Dit veld moet niet worden aangepast. De wederkerige relatie heeft de het tegenovergestelde type, indien beschikbaar, van deze relatie.</help>
+        <label>Wederkerige Relatie</label>
+        <name>ReciprocalRelationship__c</name>
+        <relationshipLabel>Relaties</relationshipLabel>
+    </fields>
+    <fields>
+        <label>Gerelateerde Contacpersoon</label>
+        <name>RelatedContact__c</name>
+        <relationshipLabel>Verborgen Relaties Lijst</relationshipLabel>
+    </fields>
+    <fields>
+        <help>Tekst formule die een zin maakt die de relatie uitlegt tussen twee contactpersonen.</help>
+        <label>Relatie Uitleg</label>
+        <name>Relationship_Explanation__c</name>
+    </fields>
+    <fields>
+        <help>Systeem veld dat aangeeft of de relatie automatisch was gemaakt als de wederhelft van en ander relatie.</help>
+        <label>_SYSTEEM: SysteemAangemaakt</label>
+        <name>SYSTEM_SystemCreated__c</name>
+    </fields>
+    <fields>
+        <help>Status van de Relatie</help>
+        <label><!-- Status --></label>
+        <name>Status__c</name>
+        <picklistValues>
+            <masterLabel>Current</masterLabel>
+            <translation>Huidig</translation>
+        </picklistValues>
+        <picklistValues>
+            <masterLabel>Former</masterLabel>
+            <translation>Oud</translation>
+        </picklistValues>
+    </fields>
+    <fields>
+        <help>Veld die beschrijft hoe de gerelateerde contactpersoon is verwant aan de contactpersoon</help>
+        <label><!-- Type --></label>
+        <name>Type__c</name>
+        <picklistValues>
+            <masterLabel>Aunt</masterLabel>
+            <translation><!-- Aunt --></translation>
```

Comment:
@njjc they weren't part of the translation file. They only had Coach which I removed since it's not a default value.